pkg/clients/AwsClient.go

```diff
@@ -617,10 +617,18 @@ func (o *AwsClient) getConfigForLambda() *aws.Config {
 
 func uploadFuncClarityCode(cfg *aws.Config, keyPath string, bucket string) error {
 	s3Client := s3.NewFromConfig(*cfg)
-	_, err := s3Client.CreateBucket(context.TODO(), &s3.CreateBucketInput{
-		Bucket:                    aws.String(bucket),
-		CreateBucketConfiguration: &s3types.CreateBucketConfiguration{LocationConstraint: s3types.BucketLocationConstraint(cfg.Region)},
-	})
+	var err error
+	if cfg.Region != "us-east-1" {
```

Comment:
There is a cons in v2 for region, you can use it instead of custom string.

Reply:
no const for us-east-1 in enums.go file